src/modeler/AutoRest.Swagger/Validation/TrackedResourceValidation.cs

```diff
@@ -138,7 +138,7 @@ private bool HandleSchema(Schema schema, Dictionary<string, Schema> definitions)
             return true;
         }
 
-        private bool listByXCheck(List<Operation> getOperations, Regex regEx, string definitionKey, Dictionary<string, Schema> definitions)
+        private bool listByXCheck(IEnumerable<Operation> getOperations, Regex regEx, string definitionKey, Dictionary<string, Schema> definitions)
```

Comment:
There are other methods in the file with similar naming convention, it would make sense to update all of them too, hence my doubts